gripper/simulator/main.cpp

```diff
@@ -70,8 +70,10 @@ int main(int argc, char** argv) {
     LOG_INIT("GRIPPER", []() -> const char* {
         return pcTaskGetName(xTaskGetCurrentTaskHandle());
     });
+    const uint32_t TEMPORARY_SERIAL = 0x103321;
```

Comment:
I think this will error if the number isn't `const uint32_t TEMPORARY_SERIAL = 0x10003321` right? Otherwise the model number will be something unexpected rather than 0 or 1?